BuyPolish/Pola/UI/ProductSearch/ScanCode/ScanCodeView.swift

```diff
@@ -86,5 +95,12 @@ final class ScanCodeView: UIView {
                 + (menuButton.bounds.height / 2)
                 - (logoButton.bounds.height / 2)
         )
+
+        NSLayoutConstraint.activate([
```

Comment:
`UIKit` poradzi sobie z tym, że część subview będzie ręcznie, a część Autolayoutem? Do tego `layoutSubviews` odpala się więcej jak raz i będziemy mieć masę nadmiarowych constraintów. Nie chciałoby Ci się przepisać layoutowania w tej klasie na `NSLayoutConstraint`? Można by to wtedy wynieść do nowej metody `setupConstraints` i odpalić ją w inicie.